server/src/modules/Question/dto/question.dto.ts

```diff
@@ -0,0 +1,8 @@
+export class QuestionDto {
```

Comment:
QuestionDto moved to Question/question.dto.ts. This is the old one.
BTW do we need contestId inside dto object?

Reply:
removed